externs/shaka/player.js

```diff
@@ -1562,6 +1563,9 @@ shaka.extern.AdvancedAbrConfiguration;
  * @property {number} rtpSafetyFactor
  *   RTP safety factor.
  *   Defaults to <code>5</code>.
+ * @property {Array<string>} includeKeys
```

Comment:
nit: The default is an empty array, and the type in the record above is non-nullable, so this should also be non-nullable.

Reply:
Done.